inputs/gin/parseFunction.go

```diff
@@ -335,11 +330,10 @@ func parseFunction(s *astra.Service, funcTraverser *astTraversal.FunctionTravers
 					currRoute, err = funcBuilder.ExpressionResult().Build(func(route *astra.Route, params []any) (*astra.Route, error) {
 						field := astra.ParseResultToField(params[0].(astTraversal.Result))
 
-						route.BodyType = "form"
-
-						route.Body = append(route.Body, astra.Param{
-							IsBound: true,
-							Field:   field,
+						route.Body = append(route.Body, astra.BodyParam{
+							ContentType: "form",
```

Comment:
Is the content type here form or `application/x-www-form-urlencoded` or `multipart/form-data`?

Reply:
Realising I put this in...

Reply:
Yeah this is me just moving around your code, tbh I have actually missed this so need to fix that anyway

Reply:
This should now be solved, there is a slight problem with the implementation but I don't think that's within the scope of this PR to fix, Its when multiple structs are embedded in a single struct.